### PR TITLE
Apple branch

### PR DESCRIPTION
**Pull request by christianotieno**
I have finished building the apple.com clone website. I am welcoming a review of my code, thanks.

### CODE REVIEW

File: stylelint.config.js [hunk 1]
```diff
@@ -78,7 +78,7 @@ module.exports = {
         "media-query-list-comma-space-after": "always-single-line",
         "media-query-list-comma-space-before": "never",
         "no-eol-whitespace": true,
-        "no-missing-end-of-source-newline": false,
+        "no-missing-end-of-source-newline": true,
```
Comment (bolah2009) [inline stylelint.config.js:81]: Is there a reason you changing this config setting? :thinking:

**Review by bolah2009 — CHANGES_REQUESTED** [review]
## Awesome work @ChristianOtieno 

### [The clone](https://rawcdn.githack.com/ChristianOtieno/apple-home-page-clone/a943ddbe7a6004c488367b64fe8e83d93204bbbd/index.html) shows good use of CSS Backgrounds and Gradients. :100: 

A few required changes: Note that for this project, we have to make sure nothing is messed up. Please see difference in the source page and the clone below: 

| [Current clone](https://rawcdn.githack.com/ChristianOtieno/apple-home-page-clone/a943ddbe7a6004c488367b64fe8e83d93204bbbd/index.html) | [Source Page](https://web.archive.org/web/20140301004610/http://www.apple.com/) |
|:-----:|:-----:|
| image  | image|




#### :recycle:  Make these changes and submit another code review request. :+1: 

#### Feel free to reach me on slack if you having issues implementing these features. :+1: 

Good luck with your next submission. :star2: 

Cheers
[Bola](https://bolabuari.com/)

###### PS: Note that comments tagged as `OPTIONAL` are not part of the minimum requirements to approve this PR

File: index.html [hunk 2]
```diff
@@ -1,11 +1,74 @@
...
+                        <button type="button" class="header-support"></button>
+                        <form class="search-bar">
+                            <input type="text" name="search-bar">
+                        </form>
+                    </div>
+                    <header class="title">
+                        <a href="#" class="top-title">What will your verse be?</a>
+                        <a href="#" id="middle-title">Everyone has something to share. See how people are using <br>
+                            iPad in their unique ways to add to the world’s story.</a>
+                        <a href="#" id="bottom-title">Watch the “Your Verse” film and explore the stories</a>
```
Comment (whiz25) [inline index.html:36]: The text on lines 35 and 36 are overlapping. please correct that.

**Review by whiz25 — CHANGES_REQUESTED** [review]
## Awesome :+1: @ChristianOtieno 
You have made some suggested changes but you have forgotten to make your clone look like the original site. Look at my suggestions below;
- [x] see my comment
- [x] give your page a width of 100%. The background image and the four links below should cover the entire width of the page.

Please make those changes and re-submit for another review.

**Review by whiz25 — APPROVED** [review]
## Awesome :clap: 
You have implemented the suggested requirements. That's good.
Please go ahead and merge. Use ```Martin Afani``` when filling the form.

**Pull request merged**